packages/ui-table/src/Table/ColHeader/index.tsx

```diff
@@ -63,10 +64,16 @@ class ColHeader extends Component<Props> {
     // eslint-disable-next-line react/require-default-props
     styles: PropTypes.object,
     /**
-     * A unique id for this column. When sortable table is in stacked layout,
-     * id is also used as option in combobox.
+     * A unique id for this column. The `id` is also used as option in combobox,
+     * when sortable table is in stacked layout,
+     * and no `stackedSortByLabel` is provided.
      */
     id: PropTypes.string.isRequired,
+    /**
+     * A custom string to display as option in the combobox (instead of `id`),
+     * when sortable table is in stacked layout.
+     */
```

Comment:
It wasn't clear for me what this property does, I had to rewrite the example code to figure out. I'd rephrase it to:
"A custom string to display as option text in the combobox (instead of using the `id` field), when sortable table is in stacked layout."

Reply:
Thank, modified it!